.github/workflows/helm.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - release-1.8
```

Comment:
since were releasing 1.8.4 and 1.9.3 were not merging this changelog to main, so helm release wont run on 4.7.2, this lets us release the helm chart